packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -546,7 +546,12 @@ class TreemapNode extends TreeNode<TreemapNode> {
       }
     }
     final separator = oneLine ? ' ' : '\n';
-    return '$displayName$separator[${prettyPrintBytes(byteSize, includeUnit: true)}]';
+    return '$displayName$separator[${prettyByteSize()}]';
+  }
+
+  String prettyByteSize() {
+    final plusSign = showDiff && byteSize > 0 ? '+' : '';
```

Comment:
what about '-' for negative diff values?

Reply:
Since we're printing out `byteSize` not `unsignedByteSize`, the negative sign is included when normally printing a negative number

Reply:
can you add a comment to explain that